src/pages/AppConfigure.vue

```diff
@@ -42,7 +42,10 @@ let currentConfig = getAppConfigRequestBody()
 
 function switchTab(tab: ConfigureTab) {
   currentTab.value = tab.type
-  router.replace({ name: `${tab.label}Settings` })
+  router.replace({
```

Comment:
Change to `router.push`. We want to user to be able to use the back button when navigating the configure pages.

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/123/commits/a646f74a4dd17b878ec757822ec6e8376a3eeb1f